tasks/noci_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+EXIT_STATE=0
+
+# tests that aren't run on CI
+
+# jasmine specs with @noCI tag
+npm run citest-jasmine --  tests/*_test.js --tags noCI || EXIT_STATE=$?
```

Comment:
done in https://github.com/plotly/dev-docs/commit/17302734858a6bc967944450490ea4ffef1757bd